test/rpc_test.py

```diff
@@ -1152,15 +1186,15 @@ def test_wait_all_workers_and_shutdown(self):
         # multiple times.
         rpc.init_rpc(
             name="worker%d" % self.rank,
-            backend=rpc.backend_registry.BackendType[dist_utils.TEST_CONFIG.rpc_backend_name],
+            backend=self.rpc_backend,
             rank=self.rank,
             world_size=self.world_size,
             rpc_backend_options=self.rpc_backend_options
         )
         from torch.distributed.rpc.api import _wait_all_workers
         # intentional call to internal _wait_all_workers.
         _wait_all_workers()
-        rpc.shutdown()
+        rpc.shutdown(graceful=False)
```

Comment:
This changes the test behavior. IIUC, this was to test calling `_wait_all_workers()` and then call `shutdown(graceful=True)`. cc @rohan-varma

Reply:
Same reason, _wait_all_workers does not support being called twice.